src/wavesurfer.js

```diff
@@ -61,6 +61,7 @@ import MediaElementWebAudio from './mediaelement-webaudio';
  * attributes.
  * @property {number} duration=null Optional audio length so pre-rendered peaks
  * can be display immediately for example.
+ * @property {number} edgeScrollWidth='5% from container edges' Optional width for edgeScroll to start
```

Comment:
this new param applies to the regions plugin, not wavesurfer core, so it shouldn't be documented here.

Reply:
Fixed it